draft-welzl-ccwg-ratelimited-increase.md

```diff
@@ -223,9 +233,39 @@ in Section 7.2.1. It is thus implicitly clear that the quoted rule only applies
 
 ### Assessment
 
-With the exception of pacing, this specification conservatively limits the growth in cwnd, similar to Cubic and SCTP. The exception for pacing in the second paragraph requires the application to notify the transport layer that it paces packets. Pacing is typically done with delays below an RTT; thus, rule #2 in {{rules}} should cover this case without the need for such a notification from the application.
+With the exception of pacing, this specification conservatively limits the growth in cwnd, similar to Cubic and SCTP.
+The exception for pacing in the second paragraph requires pacing is enabled, this is specifically taken into account. Pacing could occur over various timescales, but
```

Comment:
```suggestion
The exception for pacing in the second paragraph requires that pacing is enabled, this is specifically taken into account. Pacing could occur over various timescales, but
```

This is either a nit that you overlooked, or it's just my English: it may well be correct?  I just don't know.

Reply:
.... ~mangled ~ ... I'll untangle.